client/src/app/site/pages/meetings/pages/participants/pages/participant-import/services/participant-import.service/participant-import.service.ts

```diff
@@ -140,6 +140,10 @@ export class ParticipantImportService extends BaseUserImportService {
         const result = await this.presenter.call({
             permissionRelatedId: this.activeMeetingId,
             searchCriteria: entries.map(entry => {
+                if (entry.username) {
+                    return { username: entry.username };
+                }
+
                 const username = !!entry.username ? entry.username : `${entry.first_name} ${entry.last_name}`;
                 return { username, email: entry.email };
```

Comment:
Could probably remove the username from this second return statement then. It will be undefined anyway..